railties/lib/rails/generators/rails/app/templates/config/routes.rb.tt

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+
+  # Even single page applications require a root ("/") path:
+  # root "controller#action"
```

Comment:
```suggestion
  # Set the application's home page:
  # root "articles#index"
```

what do you think about using this example, to match the getting starting guide? https://guides.rubyonrails.org/getting_started.html#setting-the-application-home-page

Reply:
100% on board the consistency train!

Almost hit the "commit suggestion" button but paused because in the spirit of helping newcomers, it's obvious to us that `articles` is a stand-in for "your controller and action here"... but are we assuming too much?

To summarize: I like your suggestion better than mine, but we owe it to that theoretical future dev to at least have the discussion.

Reply:
My thinking was that a true beginner is hopefully (🤞 ) following the getting started guide. So when they do `rails new` and see this file, it will match what they are learning through that guide. And if they uncomment this code, it won't break their progress through the guide.

For anyone else who already knows what this does and just needs the syntax (eg. you), any example is fine.

Reply:
I'll bite!